pywb/rules.yaml

```diff
@@ -1,5 +1,8 @@
 # Default Filters
 default_filters:
+    # limit to fuzzy match prefix results
+    fuzzy_search_limit: '100'
```

Comment:
Documentation (fuzzy matching)
 
Is the limit required for upstream pywb?

Reply:
Yes, this was to prevent unbounded prefix queries, which can pull significant results on a large archive.

Reply:
Makes sense thanks for clarifying!